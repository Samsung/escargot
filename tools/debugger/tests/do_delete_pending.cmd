b do_delete_pending.js:21
b pending1
y
b pending2
y
list
delete pending
list
b pending3
y
delete all
list
quit
