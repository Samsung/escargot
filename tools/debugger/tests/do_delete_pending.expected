Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_delete_pending.js:20
(escargot-debugger) b do_delete_pending.js:21
Breakpoint 1 at tools/debugger/tests/do_delete_pending.js:21
(escargot-debugger) b pending1
No breakpoint found, do you want to add a pending breakpoint? (y or [n])
Pending breakpoint 2 at pending1()
(escargot-debugger) b pending2
No breakpoint found, do you want to add a pending breakpoint? (y or [n])
Pending breakpoint 3 at pending2()
(escargot-debugger) list
=== Active breakpoints ===
 1: tools/debugger/tests/do_delete_pending.js:21
=== Pending breakpoints ===
 2: pending1() (pending)
 3: pending2() (pending)
(escargot-debugger) delete pending
(escargot-debugger) list
=== Active breakpoints ===
 1: tools/debugger/tests/do_delete_pending.js:21
(escargot-debugger) b pending3
No breakpoint found, do you want to add a pending breakpoint? (y or [n])
Pending breakpoint 4 at pending3()
(escargot-debugger) delete all
(escargot-debugger) list
No breakpoints
(escargot-debugger) quit
Connection closed.
